Editor/Windows/CinemachineSettings.cs

```diff
@@ -323,12 +323,23 @@ private static void OnGUI()
                 GUILayout.EndScrollView();
             }
 
+            // set label width, so text is not cut for Toggles
+            float originalValue = EditorGUIUtility.labelWidth;
+            EditorGUIUtility.labelWidth = 180f;
+            {
 #if CINEMACHINE_UGUI
-            // Storyboard global mute
-            CinemachineStoryboardMute.Enabled = EditorGUILayout.Toggle(
-                new GUIContent("Storyboard Global Mute", "If checked, all storyboards are globally muted."), 
-                CinemachineStoryboardMute.Enabled);
+                // Storyboard global mute
+                CinemachineStoryboardMute.Enabled = EditorGUILayout.Toggle(
+                    new GUIContent("Storyboard Global Mute", "If checked, all storyboards are globally muted."),
+                    CinemachineStoryboardMute.Enabled);
 #endif
+                // Editable Game Window Guides 
+                CinemachineScreenComposerGuidesGlobalEditable.Enabled = EditorGUILayout.Toggle(
+                    new GUIContent("Editable Game Window Guides", "If checked, game window guides are editable in play " +
```

Comment:
I think it should be "Draggable" instead of "Editable", because you can still edit it from the inspector.